src/tests/ftest/util/daos_racer_utils.py

```diff
@@ -45,8 +45,11 @@ def __init__(self, path, host, dmg_config=None):
         # Number of seconds to run
         self.runtime = FormattedParameter("-t {}", 60)
 
-        if dmg_config:
-            self.dmg_config = FormattedParameter("-n {}", dmg_config)
+        if dmg:
+            self.dmg_config = FormattedParameter("-n {}", dmg.yaml.filename)
+            hostlist = []
+            hostlist.append(host)
+            dmg.copy_certificates(get_log_file("daosCA/certs"), hostlist)
```

Comment:
This can be simplified to use less lines:
```suggestion
            dmg.copy_certificates(get_log_file("daosCA/certs"), [host])
```

Reply:
It should probably use `self.host` as well:
```
            dmg.copy_certificates(get_log_file("daosCA/certs"), [self.host])
```

Reply:
Good idea. Done.